packages/addresses/src/index.ts

```diff
@@ -0,0 +1,90 @@
+import { config as mainnetConfig } from '@oasisdex/dma-deployments/configs/mainnet.conf'
+import { config as optimismConfig } from '@oasisdex/dma-deployments/configs/optimism.conf'
+import { config as goerliConfig } from '@oasisdex/dma-deployments/configs/goerli.conf'
+
+import { DeploymentConfig, SystemKeys } from '@oasisdex/dma-deployments/types/deployment-config'
+import { Network } from '@oasisdex/dma-common/utils/network'
+
+type DeployedNetworks = Network.MAINNET | Network.OPT_MAINNET | Network.GOERLI
+// TODO: Make address entries type safe after https://github.com/OasisDEX/oasis-earn-sc/pull/249/files is merged
+export type Addresses = Record<DeployedNetworks, Record<SystemKeys, any>>
+
+export const ADDRESSES: Addresses = {
+  [Network.MAINNET]: {
```

Comment:
What do you think about exporting `[Network.MAINNET]` again but as `[Network.HARDHAT]`? Just for the ease of use.

Reply:
But Hardhat could be using Optimism or Mainnet addresses depending on the fork

Reply:
Right.